src/packet-ffxiv.c

```diff
@@ -145,8 +150,15 @@ static int dissect_frame(tvbuff_t *tvb, packet_info *pinfo, proto_tree *tree, vo
     next_tvb = tvb_new_subset_remaining(tvb, FRAME_HEADER_LEN);
   }
 
-  tcp_dissect_pdus(next_tvb, pinfo, frame_tree, TRUE, BLOCK_HEADER_LEN,
-    get_message_length, dissect_message, data);
+  offset = 0;
```

Comment:
Thanks, it just makes things a little clearer. We could also just use offset as the message offset since the original offset is only used for `build_frame_header` which can take it as `0` to begin with. The offset is incremented by the length of the frame header but it doesn't get used. This may just be code cleanup so feel free to ignore it, but renaming the TVB would be good.